scripts/safari.js

```diff
@@ -309,7 +311,7 @@ function Safari() {
     };
     var finderItems = {
         crown: 1, honey: 1,
-        rare: 5, recharge: 8, rock: 16, bait: 16, pearl: 16, stardust: 8, luxury: 16, gacha: 13,
+        rare: 5, recharge: 8, spy: 32, rock: 16, bait: 16, pearl: 16, stardust: 8, luxury: 16, gacha: 13,
```

Comment:
You'll need to add a case to itemfinder if I recall. The default is for nothing found, since everything has a different message. Also the rate seems high cause it drops "nothing" from 80% to like 75%.